packages/local-cli/bundle/buildBundle.js

```diff
@@ -10,8 +10,6 @@
 import type { ContextT } from '../core/types.flow';
 import type { CommandLineArgs } from './bundleCommandLineArgs';
 
-('use strict');
```

Comment:
You probably shouldn't remove `'use strict'` if you want your scripts to run in strict mode as you're not using ES6 modules.

Reply:
This is what's used: `@babel/plugin-transform-strict-mode`. Does it matter which sourceType we use when having that?

Reply:
Yea, I requested it :D glad you're up to date now :)